multinode-demo/start_nodes.sh

```diff
@@ -41,25 +40,45 @@ leader=
 for ip_addr in "${ip_addr_array[@]}"; do
   echo "$ip_addr"
 
+  ssh-keygen -R "$ip_addr"
+  ssh-keyscan "$ip_addr" >>~/.ssh/known_hosts
+
+  ssh "$remote_user"@"$ip_addr" 'mkdir ~/.ssh'
+
+  ssh -n -f "$remote_user"@"$ip_addr" "sudo service sshguard stop"
```

Comment:
Let's add a TODO comment here to avoid killing sshguard (filing an issue works too), as well as documenting why in a comment.  For now this is ok, but in general we don't want to do be doing this IMO